test/styles/bibtex.js

```diff
@@ -0,0 +1,72 @@
+const assert = require('assert');
+const bibtexmisc = require('../../model/styles/bibtexMisc');
+const biblatexSoftware = require('../../model/styles/biblatexSoftware');
+
+const oneAuthorSourceData = require('../mockSourceData/allFieldsOneAuthor');
+const threeAuthorsSourceData = require('../mockSourceData/allFieldsThreeAuthors');
+
+describe('BibTexMiscStyle', () => {
+  describe('format', () => {
+    it('Month field, one author', () => {
+      let formattedString = bibtexmisc.format(oneAuthorSourceData);
+      assert.equal(formattedString.includes("month = {11},"),true);
+    });
+    
+    it('Title field one author', () => {
+      let formattedString = bibtexmisc.format(oneAuthorSourceData);
+      assert.equal(formattedString.includes("title = {Red Barchetta},"),true);
+    });
+    
+    it('Author field, one author', () => {
+      let formattedString = bibtexmisc.format(oneAuthorSourceData);
+      assert.equal(formattedString.includes("author = {Lee, Geddy},"),true);
+    });
+    
+    it('Note field, one author', () => {
+      let formattedString = bibtexmisc.format(oneAuthorSourceData);
+      assert.equal(formattedString.includes("note = {Version: 1.0.0, URL: http://myUnclesFarm.com}"),true);
+    }); 
+    //Three Authors
+    it('Author field, three authors', () => {
+      let formattedString = bibtexmisc.format(threeAuthorsSourceData);
+      assert.equal(formattedString.includes("author = {Lee, Geddy and Lifeson, Alex and Peart, Neil},"),true);
+    });
+
+  });
+});
+
+describe('BibLatexSoftwareStyle', () => {
+    describe('format', () => {
+
+      it('Month field, one author', () => {
+        let formattedString = biblatexSoftware.format(oneAuthorSourceData);
+        assert.equal(formattedString.includes("month = {11},"),true);
+      });
+    
+      it('Title field one author', () => {
```

Comment:
If you mean the tests, @nokeeo's tests were also not comma separated, so I just followed suit.